src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -88,7 +82,8 @@ def start(scenario, settings, events, aggregator_device_mapping):
             "max_panel_power_W": settings.get('max_panel_power_W',
                                               ConstSettings.PVSettings.MAX_PANEL_OUTPUT_W),
             "grid_fee_type": settings.get('grid_fee_type', GlobalConfig.grid_fee_type),
-            "external_connection_enabled": True
+            "external_connection_enabled": True,
```

Comment:
We should think of setting the default value here to False and control this via the d3a-web for the reason you mentioned above in cli.py

Reply:
Yes, I agree, will use `False` as default